developer/kmcompx/src/meson.build

```diff
@@ -0,0 +1,102 @@
+#
+# Keyman is copyright (C) SIL International. MIT License.
+#
+# Cross platform build script to compile kmcompx
+#
+
+defns = ['-DKMN_KBP_EXPORTING']
+version_res = []
+
+# TODO: move these shared defines to a resources/meson folder?
+if compiler.get_id() == 'gcc' or compiler.get_id() == 'clang'
+  warns = [
+     '-Wall',
+     '-Wctor-dtor-privacy',
+     '-Wdouble-promotion',
+     '-Wendif-labels',
+     '-Wextra',
```

Comment:
Meson's default options: `warning_level=1` will already set `-Wall`, level 2 adds `-Wextra`, and it is preferred to set it to `warning_level=2` in your project() options if you want `-Wextra` rather than hardcoding it in the build file (it will handle MSVC for free for you)